dart/.packages

```diff
@@ -1,2 +1,17 @@
-# Generated by pub on 2016-08-20 13:39:08.695546.
+# Generated by pub on 2019-12-01 22:05:57.410937.
+analysis_server_lib:file:///Users/brettmorgan/.pub-cache/hosted/pub.dartlang.org/analysis_server_lib-0.1.8/lib/
+args:file:///Users/brettmorgan/.pub-cache/hosted/pub.dartlang.org/args-1.5.2/lib/
+charcode:file:///Users/brettmorgan/.pub-cache/hosted/pub.dartlang.org/charcode-1.1.2/lib/
+cli_util:file:///Users/brettmorgan/.pub-cache/hosted/pub.dartlang.org/cli_util-0.1.3+2/lib/
+collection:file:///Users/brettmorgan/.pub-cache/hosted/pub.dartlang.org/collection-1.14.12/lib/
+intl:file:///Users/brettmorgan/.pub-cache/hosted/pub.dartlang.org/intl-0.15.8/lib/
+logging:file:///Users/brettmorgan/.pub-cache/hosted/pub.dartlang.org/logging-0.11.3+2/lib/
+meta:file:///Users/brettmorgan/.pub-cache/hosted/pub.dartlang.org/meta-1.1.8/lib/
+path:file:///Users/brettmorgan/.pub-cache/hosted/pub.dartlang.org/path-1.6.4/lib/
+pedantic:file:///Users/brettmorgan/.pub-cache/hosted/pub.dartlang.org/pedantic-1.8.0+1/lib/
+source_span:file:///Users/brettmorgan/.pub-cache/hosted/pub.dartlang.org/source_span-1.5.5/lib/
+string_scanner:file:///Users/brettmorgan/.pub-cache/hosted/pub.dartlang.org/string_scanner-1.0.5/lib/
+term_glyph:file:///Users/brettmorgan/.pub-cache/hosted/pub.dartlang.org/term_glyph-1.1.0/lib/
+tuneup:file:///Users/brettmorgan/.pub-cache/hosted/pub.dartlang.org/tuneup-0.3.6+1/lib/
+yaml:file:///Users/brettmorgan/.pub-cache/hosted/pub.dartlang.org/yaml-2.2.0/lib/
 mal:lib/
```

Comment:
Please drop this change from the push. It contains local file paths.

Reply:
done